packages/backend/migrations/20230201133042_create_ledger_accounts_table.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema.createTable('ledgerAccounts', function (table) {
+    table.uuid('id').notNullable().primary()
+    table.uuid('accountRef').notNullable().unique().index()
+
+    table.uuid('assetId').notNullable()
+    table.foreign('assetId').references('assets.id')
+
+    table
+      .enu('type', [
+        'LIQUIDITY',
+        'LIQUIDITY_ASSET',
+        'LIQUIDITY_PEER',
+        'LIQUIDITY_INCOMING',
+        'LIQUIDITY_OUTGOING',
+        'SETTLEMENT'
+      ])
+      .notNullable()

```

Comment:
Storing the `type` directly in the DB makes the ledger accounts easier to keep track of. Currently, when we create asset/ledger accounts, we create a LIQUIDITY_ASSET account with the `id` of the tigerbeetle account id being equal to the `asset.id`, and also a `SETTLEMENT` account, where the tigerbeetle account id equals the `asset.ledger`, meaning a `smallint` number:
https://github.com/interledger/rafiki/blob/ec2f24d5f6d17c76faf5e0a781ad257baef95496/packages/backend/src/asset/service.ts#L66-L78

https://github.com/interledger/rafiki/blob/ec2f24d5f6d17c76faf5e0a781ad257baef95496/packages/backend/src/accounting/service.ts#L129-L173

Now, we would be able to create these LedgerAccounts using `ledgerAccount.accountRef = assetId` and `type = LIQUIDITY_ASSET | SETTLEMENT`, and as a result we can just find the ledger accounts via { assetId, type }.